rfc/4/orientation.yml

```
@@ -1,39 +1,330 @@
+default_prefix: ngff
+
+created_on: "2025-05-29"
+created_by: orcid:0000-0003-1666-5421
+contributors: orcid:0000-0001-9475-3756
+
+classes:
+  # Class pertaining to OME-NGFF axes definition
+  # Example of individual axis object: { "name": "z", "type": "space", "unit": "micrometer"}
+  Axes:
+    attributes:
+      axes:
+        description: >
+          A list of axes. Although serialized as list, it MUST be dealt with as being a set as in the name of each axis
+          MUST be unique. Furthermore, if the attribute orientation is defined for one axis of type space, it 
+          MUST be defined for all the axes of type space. In this case, the type of each orientation MUST be the same
+          and the value MUST be unique.
+        multivalued: true
+        any_of:
+          - range: SpaceAxis
+          - range: TimeAxis
+          - range: ChannelAxis
+
+  Axis:
+    abstract: true
+    attributes:
+      name:
+        required: true
+        range: AxesNames
+      type:
+        range: AxisType
+        required: true
+
+
+  ChannelAxis:
+    is_a: Axis
+    slot_usage:
+      name:
+        range: string
+        equals_string: c
+        ifabsent: string(c)
+      type:
+        range: string
+        equals_string: channel
+        ifabsent: string(channel)
+    defining_slots:
+      - name
+      - type
+
+  SpaceAxis:
+    is_a: Axis
+    slot_usage:
+      name:
+        range: SpaceAxesNames
+      type:
+        range: string
+        equals_string: space
+        ifabsent: string(space)
+    attributes:
+      unit:
+        description: >
+          Physical unit for spatial measurement along the axis, selected from a standardized list of distance units 
+          (e.g., micrometer, nanometer).
+        range: SpaceUnit
+        required: true
+      orientation:
+        description: The direction of an axis of type space.
+        any_of:
+          - range: AnatomicalOrientation
+        required: false
+    defining_slots:
+      - name
+      - type
+      - unit
+
+  TimeAxis:
+    is_a: Axis
+    slot_usage:
+      name:
+        range: string
+        equals_string: t
+        ifabsent: string(t)
+      type:
+        range: string
+        equals_string: time
+        ifabsent: string(time)
+    attributes:
+      unit:
+        description: >
+          Temporal unit of measurement for the axis, selected from standardized time units (e.g., second, minute, hour).
+        range: TimeUnit
+        required: true
+    defining_slots:
+      - name
+      - type
+      - unit
+
+  AnatomicalOrientation:
+    attributes:
+      type:
+        range: string
+        required: true
+        equals_string: "anatomical"
+      value:
+        range: AnatomicalOrientationValues
+        required: true
+
+
+slots:
+  name:
+    description: The name identifier for an axis
+    range: AxesNames
+    required: true
+  type:  
+    description: The type category of an axis
+    range: AxisType
+    required: true

```

```suggestion
slots:
  name:
  type:  
```
This would make the slots generally reusable throughout the specification.